package.json

```diff
@@ -184,6 +184,7 @@
     "react-router-dom": "^5.3.3",
     "react-table": "^7.8.0",
     "react-virtualized-auto-sizer": "^1.0.5",
+    "react-virtuoso": "^4.7.1",
```

Comment:
required to support windowed rendering in a grouped lists. `react-window` supports grouped lists, but the groups are windowed not the list items, i.e react-window is good for lots of small groups, but bad for few huge groups.